src/org/opendatakit/briefcase/util/ServerUploader.java

```diff
@@ -204,7 +212,14 @@ public boolean uploadFormAndSubmissionFiles(List<FormStatus> formsToTransfer) {
       File briefcaseFormMediaDir = FileSystemUtils.getMediaDirectoryIfExists(briefcaseLfd.getFormDirectory());
 
       boolean outcome;
-      outcome = uploadForm(formToTransfer, briefcaseFormDefFile, briefcaseFormMediaDir);
+      boolean existsAlready = http.execute(server.containsFormQuery(formToTransfer.getFormDefinition().getFormId()));
+      if (forceSendBlank || !existsAlready)
+        outcome = uploadForm(formToTransfer, briefcaseFormDefFile, briefcaseFormMediaDir);
+      else {
+        formToTransfer.setStatusString("Skipping form upload to remote server because it already exists", true);
+        EventBus.publish(new FormStatusEvent(formToTransfer));
+        outcome = true;
+      }
```

Comment:
Below this: Bitwise **and** with booleans‽  Also:
![screen shot 2018-05-15 at 12 10 13 pm](https://user-images.githubusercontent.com/55099/40078330-12e78096-5839-11e8-8d71-bd97a5dc30bb.png)